framework/material/Form.kt

```diff
@@ -491,16 +497,21 @@ object Form {
     fun MultilineTextInput(
```

Comment:
By the way, the existing usage of this (Finder/Replacer) seems to be buggy. Hitting the enter key, or shift-enter, doesn't insert a newline. We should investigate and/or raise an issue.